tinyagentos/routes/memory_management.py

```
@@ -132,3 +192,107 @@ async def agent_memory_config_put(request: Request, name: str):
+        return JSONResponse({"error": str(exc)}, status_code=500)
+
+
+@router.post("/api/memory/recipes/{recipe_id}/apply")
+async def memory_recipes_apply(request: Request, recipe_id: str):
+    """Apply a recipe as the global default or to a specific agent.
+
+    Body (all optional): ``{"agent": "<name>"}``
+    """
+    try:
+        body = await request.json()
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)